search-app/docs/api.md

```diff
@@ -1578,7 +1578,7 @@ Find collections that match all of the 'project' param values
 This supports `pattern`, `ignore_case` and option `and`.
 
 Find collections matching 'consortium' param value
-CWIC, FEDEO, CEOS, EOSDIS
```

Comment:
why remove example values, it's going to be hard enough to explain what a consortium is, having these names would help.

Reply:
This is just to fix the bug Matt mentioned in the documentation. Somehow the example in the line below it didn't show the escaped bracket, even though in the code the escape was there. so I just cleaned up the line and made it exactly the same as project example above.  I didn't really want to put these values there I forgot why I did. it's not complete and not directly describing the example below.

Reply:
I see there are some URLs with example values, that should work for my needs